external/hbase/src/test/scala/org/apache/gearpump/external/hbase/HBaseSinkSpec.scala

```diff
@@ -38,6 +39,7 @@ class HBaseSinkSpec extends PropSpec with PropertyChecks with Matchers {
     val hbaseSink = HBaseSink(htable)
     hbaseSink.insert(put)
     verify(htable).put(put)
+    */
```

Comment:
When you construct a DataSinkProcessor, it will serialize the datasink to the UserConfig, which will fail if contains a HTable.